src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -305,7 +305,12 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
         this._eventManager.listen(this._videoElement, Html5EventType.ABORT, () => this._clearHeartbeatTimeout());
         this._eventManager.listen(this._videoElement, Html5EventType.SEEKED, () => this._syncCurrentTime());
         // Sometimes when playing live in safari and switching between tabs the currentTime goes back with no seek events
-        this._eventManager.listen(window, 'focus', () => setTimeout(() => this._syncCurrentTime(), BACK_TO_FOCUS_TIMEOUT));
+        this._eventManager.listen(window, 'focus', () =>
+          setTimeout(() => {
+            this._videoElement.currentTime = Math.max(0, this._videoElement.currentTime - 0.1);
```

Comment:
when can this happen? when currentTime is 0?

Reply:
Yes, just wanted to have a protection in case of an edge case, it was suggested by Roy B.

Reply:
Ok, so you can donit with simple tenrary currenTime > 0.1 ? currentTime-0.1 : 0
What happens with live? Will this still be correct?

Reply:
Done,
And it is working with Live entries as well.